### DISCUSSION TITLE
Boot up with systemd-boot

### DISCUSSION THREAD

**Discussion by ZBookCMD** [Sugaryyyy/FemboyOS#10] [Show and tell]
For the last 10 days or so I've been working on running this distribution on a real machine owo
I have already compiled my own kernel for it and adapted most of for myself
And.. just tried to boot from the USB device, but that dont see my drive and panics
Transfered all files to HDD in PC, set "root=/dev/sddX" andd **that works :з**

![proofs >w<](https://github.com/user-attachments/assets/16a5004a-7f94-4e56-b0ab-29245671ff7e)

I’m thinking of continuing the work, but under the name miniSYS.. and this is my first and last discussion in this repo

**Comment by Sugaryyyy** [comment]
Nice!~ Glad you got it working! I'm guessing the issue with the kernel panic on boot with a USB drive was because of Ventoy, because I haven't implemented Ventoy support in PR3, it was in PR4 but it's uh... kind of gone. I'll probably come back to this OS someday, but anyway, good luck on the distro :3

**Reply by ZBookCMD** [reply]
Thanks uwu
I still believe that FemboyOS will become popular, and you will be able to promote it. As for me, the idea is original and relevant ;з

Regarding the kernel panic when booting from a USB device, this is more of an init problem..
The kernel simply does not have time to initialize USB devices, which is why it does not see the disk and goes into panic.. 
Just my guess, although that's what I had... owo"
